spec/src/main/asciidoc/configuration.asciidoc

```diff
@@ -38,7 +38,7 @@ Any dynamic changes afterwards will be ignored until  the application restarting
 
 This specification defines the annotations: `@Asynchronous`, `@Bulkhead`, `@CircuitBreaker`, `@Fallback`, `@Retry` and `@Timeout`.
 Each annotation except `@Asynchronous` has parameters.
-All of the parameters are configurable.
+All of the parameters are configurable. 
```

Comment:
Is it intended that your editor add a space after the EOL ?

Reply:
nope. removed it.